lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -81,6 +82,7 @@ sub register {
   $self->minify($config->{minify} // $app->mode ne 'development');
   $self->out_dir($self->_build_out_dir($config, $app));
   $self->base_url($config->{base_url}) if $config->{base_url};
+  $self->source_paths($config->{source_paths}) if $config->{source_paths};
```

Comment:
I want a method which automatically expands relative ```source_paths```, so you don't have to call ```$app->home->rel_dir('public')``` yourself. That way you can do this:

      $app->plugin("AssetPack" => { source_paths => ['assets'] });

Moving the assets out of "public" is a great idea, since it will make it easier for morbo to ```--watch``` directories.